src/lib/info_tree.rs

```diff
@@ -52,11 +53,22 @@ const REL_SUBTREES_SIZE: usize = 3;
 // Size of the node type vector.
 const NODE_TYPE_SIZE: usize = 3;
 
-enum PathIdx {
+/// Constants related to forward and reverse paths through trees.
+pub enum PathIdx {
+    #[allow(missing_docs)]
```

Comment:
If you push this `#[allow]` on the `enum` does it suppress the warning for each entry or ...?

Reply:
Yes, it doesn't affect the rest of the file.